plugins/obs-ffmpeg/jim-nvenc.c

```diff
@@ -997,8 +1005,9 @@ static bool init_encoder(struct nvenc_data *enc, bool hevc,
 
 		blog(LOG_WARNING, "[jim-nvenc] init_specific_encoder failed, "
 				  "trying again without Psycho Visual Tuning");
+		reinit_session(enc);
 		if (!init_specific_encoder(enc, hevc, settings, encoder, bf,
-					   psycho_aq)) {
+					   false)) {
```

Comment:
can confirm that I had the same problem. Current versions of the SDK also allow for testing for `psycho_aq` support beforehand

Reply:
What would that test look like?

Reply:
hi, do you mean `nv_get_cap_h264(enc, NV_ENC_CAPS_SUPPORT_LOSSLESS_ENCODE)` for testing or something else?

> Can you work NVENC into the PR/commit titles? obs-ffmpeg has many encoders, so it's nice to see at a glance which one.
> 
> Also, there's an unused variable in init_specific_encoder. Can you remove that while you're here?
> 
> Thanks.

it done.
thanks for your revsion.

Reply:
Resolving this because it's not an action item.

Reply:
do you mean `nv_get_cap_h264(enc, NV_ENC_CAPS_SUPPORT_LOSSLESS_ENCODE)` for testing support or something else?
could you please give more information.
thx. @theHamsta

Reply:
@jpark37 yes, this nvEncGetEncodeCaps/NV_ENC_CAPS_SUPPORT_TEMPORAL_AQ is what I meant. It's already checked in `init_encoder_h264` and ignored if not supported

Reply:
K, thanks. I think I'm happy with where this PR is for the time being.